projects/goharbor/harbor/docker/linux/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
```

Comment:
What is this image used for?  It looks like it has the chart in it?  Is that a cli?